etc/env/example.env

```diff
@@ -1,10 +1,19 @@
 # CanDIGv2 Global Environment Variables
 # - - -
 
+MODE=dev
+
 # site options
-CANDIG_MODULES=weavescope portainer consul traefik logging monitoring minio mc drs-server htsget-server igv-js wes-server chord-metadata datasets cnv_service rnaget candig_server federation-service authorization jupyter
+CANDIG_MODULES=authz weavescope portainer consul traefik logging monitoring minio mc drs-server htsget-server igv-js wes-server chord-metadata datasets cnv_service rnaget candig_server federation-service authorization
 # options are [<ip_addr>, <url>]
+
 CANDIG_DOMAIN=candig.local
+CANDIG_AUTH_DOMAIN=candigauth.local
```

Comment:
why does the auth domain need to be different from the host domain?

Reply:
it doesn't have to be different